app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,56 @@
+class CommentsController < ApplicationController
+    def index
+        @post = Post.find(params[:post_id])
+        @comments = @post.comments.all
+        redirect_to post_path(@post)
+    end
+
+    # new
+    def new
+        @post = Post.find(params[:post_id])
+        @comment = Comment.new
+    end
+
+    # create
+    def create
+        @post = Post.find(params[:post_id])
+        @comment = @post.comments.create(comment_params)
+
+        redirect_to post_comments_path(@post, @comment)
+    end
+
+    #show
+    def show
+        @post = Post.find(params[:post_id])
+        @comment = Comment.find(params[:id])
+    end
+
+    # edit
+    def edit
+        @post = Post.find(params[:post_id])
+        @comment = Comment.find(params[:id])
+    end
+
+    # update
+    def update
+        @post = Post.find(params[:post_id])
+        @comment = Comment.find(params[:id])
+        @comment.update(comment_params)
+
+        redirect_to post_comments_path(@post, @comment)
+    end
+
+    # destroy
+    def destroy
+        @post = Post.find(params[:post_id])
+        @comment = Comment.find(params[:id])
+        @comment.destroy
+
+        redirect_to post_comments_path( @post )
+    end
+
+    private
+    def comment_params
+        params.require(:comment).permit(:response, :created_at, :updated_at, :post_id)
```

Comment:
`created_at` and `updated_at` are timestamps, so they'll be generating automatically absent of user input.

Do you need to include `post_id` in strong params? Isn't that something you can assign regardless of user input?